core/templates/pages/about-page/about-page.component.html

```diff
@@ -49,19 +49,19 @@ <h2 [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_HEADING' | translate"></h2>
               <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
                 <img [src]="getStaticImageUrl('/general/creator-create-exp.png')" alt="">
                 <h3 [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE' | translate"></h3>
-                <p style="margin-top: -10px;" [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT' | translate">
+                <p class="oppia-about-page" [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT' | translate">
```

Comment:
This does not seem like the right class name. Please use one that more specifically describes this element -- "oppia-about-page" suggests something that relates to the entire page.

Reply:
Done.